### PR TITLE
chore: stop supporting helm v2

### PR DESCRIPTION
**Pull request by sumo-drosiek**
### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

**Comment by aboguszewski-sumo** [comment]
We still have some incorrect dates in this file.
`3.0` was released on January 20th
`3.1` was released on February 9th
`3.2` was released on February 16th
`3.3` was released on March 1st
So, we should drop support only for v2 here.

**Comment by sumo-drosiek** [comment]
January was correct 20 January plus 6 months is 20 July

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Sumo Logic Helm Chart Version
 | [v3.3](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.3/docs/README.md)     | deprecated / supported until 2023-09-27 |
 | [v3.2](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.2/docs/README.md)     | deprecated / supported until 2023-09-01 |
 | [v3.1](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.1/docs/README.md)     | deprecated / supported until 2023-08-09 |
-| [v3.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.0/docs/README.md)     | deprecated / supported until 2023-07-20 |
-| [v2.19](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v2.19/deploy/README.md) | deprecated / supported until 2023-07-20 |
+| [v3.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.0/docs/README.md)     | deprecated / unsupported                |
```
Comment (sumo-drosiek) [inline README.md:30]:
```suggestion
| [v3.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.0/docs/README.md)     | deprecated / supported until 2023-08-09 |
```

**Review by aboguszewski-sumo — APPROVED** [review]
@sumo-drosiek please review after my push 😅 and merge if everything is ok

**Comment by sumo-drosiek** [comment]
LGTM :heart:

**Pull request merged**